packages/react/src/components/progress/progress.tsx

```diff
@@ -0,0 +1,38 @@
+import { c, classy, m, ProgressProps as BaseProps } from '@onfido/castor';
+import React from 'react';
+
+export const Progress = ({
+  value = 0,
+  max = 100,
+  size = 'regular',
+  hideLabel,
```

Comment:
I wonder if we should follow `hide` instead of `show`? That is, should this rather be `showLabel` that defaults to `true`?

This would also make JSX (IMO) nicer, where you do `showLabel && ...` instead of `!hideLabel && ...`.

@rabelloo what do you think?

Reply:
I'm fine also with `showLabel` instead of `hide`, I agree that `show` wold make the JSX nicer, but I think `hide` has a better syntax for the user:

`<Progress value={50} hideLabel />`

instead of

`<Progress value={50} showLabel={false} />`

I'm fine with either, so let me know what you think and I change it.

Reply:
That's also a valid point.

Probably let's leave this for now as it is.

Reply:
I would prefer if no label was the default and `<Progress autoLabel />` would generate an automatic label out of the value.

However the requirement is that the auto-label is "on" by default, so...

I prefer `<Progress value={50} hideLabel />` too